tools/autograd/templates/Functions.h

```diff
@@ -17,6 +17,7 @@ namespace torch { namespace autograd { namespace generated {
 using at::Scalar;
 using at::Tensor;
 using at::IntArrayRef;
+using at::ArrayRef;
```

Comment:
is this necessary?  I don't see any generated code that needs this.

Reply:
Oh, this might have only been needed for the next diff in the stack, which uses `ArrayRef<double>`